src/providers/https.ts

```diff
@@ -132,6 +138,28 @@ export class TaskQueueBuilder {
       convertInvoker
     );
 
+    func.__endpoint = {
+      platform: 'gcfv1',
+      ...optionsToEndpoint(this.depOpts),
+      taskQueueTrigger: {},
+    };
+    copyIfPresent(func.__endpoint.taskQueueTrigger, this.tqOpts, 'retryConfig');
+    copyIfPresent(func.__endpoint.taskQueueTrigger, this.tqOpts, 'rateLimits');
+    convertIfPresent(
+      func.__endpoint.taskQueueTrigger,
+      this.tqOpts,
+      'invoker',
+      'invoker',
+      convertInvoker
```

Comment:
Still think we should be using the invoker shorthand in the manifest

Reply:
convertInvoker keeps it as shorthand if passed in a shorthand invoker.